packages/terra-i18n-plugin/src/I18nAggregatorPlugin.js

```diff
@@ -0,0 +1,61 @@
+import fs from 'fs';
+import path from 'path';
+import mkdirp from 'mkdirp';
+
+import supportedLocales from './i18nSupportedLocales';
+
+function generateTranslationFile(language, messages) {
+  return `import { addLocaleData } from 'react-intl';
+import localeData from 'react-intl/locale-data/${language.split('-')[0]}';
+
+addLocaleData(localeData);
+
+const messages = ${JSON.stringify(messages, null, 2)};
+
+module.exports = {
+  areTranslationsLoaded: true,
+  locale: '${language}',
+  messages,
+};`;
+}
+
+function getDirectories(srcPath) {
+  return fs.readdirSync(srcPath).filter(file => fs.statSync(path.join(srcPath, file)).isDirectory());
+}
+
+function apply(options) {
```

Comment:
Renamed to aggregateTranslations [here](https://github.com/cerner/terra-core/pull/272/commits/b478c09c858a6e60c1a28bea149790077150ffaa#diff-4417b6331e7583b396c941b65e5e4246R67)